frontend/src/components/terminal_player/player.ts

```diff
@@ -44,7 +44,6 @@ export default class extends EventEmitter {
       rows: Math.max(this.content.header.height, 30),
       disableStdin: true,
       fontFamily: "monospace",
-      bellStyle: "none",
```

Comment:
removed what was previously an optional param (and the "none" option didn't even do anything). The param has since been deprecated in version 5